cmd/dep/status.go

```diff
@@ -793,12 +793,23 @@ func collectConstraints(ctx *dep.Ctx, p *dep.Project, sm gps.SourceManager) (con
 			// Get project constraints.
 			pc := manifest.DependencyConstraints()
 
+			imports, err := projectImports(sm, proj)
+			if err != nil {
+				errCh <- errors.Wrapf(err, "error listing imports used in %s", proj.Ident().ProjectRoot)
```

Comment:
If we send this error to the existing error channel, we might end up overflowing the error channel. I would recommend creating another error channel for collecting these errors separately, similar to `errListPkgCh` and `errListVerCh` in `runStatusAll()`.

Reply:
I probably won't need to error after refactoring for the use case above 😄